usr.sbin/devinfo/devinfo.c

```diff
@@ -31,46 +32,127 @@
  * Print information about system device configuration.
  */
 
-#include <sys/types.h>
+#include <sys/param.h>
 #include <err.h>
 #include <errno.h>
+#include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <libxo/xo.h>
```

Comment:
style(9) also suggests a blank line after unistd.h
And after sys/param.h. (but if that's the only issue, I'll just fix it before pushing)